vectors/no_resizable_vec.rs

```diff
@@ -111,233 +96,189 @@ impl<T> NoResizableVec<T> {
             None
         } else {
             self.len -= 1;
-            unsafe { Some(ptr::read(self.ptr().add(self.len))) }
-        }
-    }
-
-    pub fn find(& self, elem: &T) -> Option<usize>
-        where  T: Ord, {
-        let mut i = 0;
-        while i < self.len {
-            if self.index(i).cmp(elem) ==  Ordering::Equal {
-                return Some(i)
+            unsafe {
+                let end: *mut T = self.buf.ptr.as_ptr().add(self.len);
+                Some(end.read())
```

Comment:
what is the difference between `ptr::read` vs `.read()`

Reply:
its the same. `ptr::read` has the `src` pointer as its first argument